cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Boundary.java

```diff
@@ -70,6 +71,12 @@ public void addLineAtNode(PropertyBag line, String node) {
         lines.add(line);
     }
 
+    public void addEquivalentInjectionAtNode(PropertyBag equivalentInjection, String node) {
+        List<PropertyBag> equivalentInjections;
+        equivalentInjections = nodesEquivalentInjections.computeIfAbsent(node, ls -> new ArrayList<>(2));
+        equivalentInjections.add(equivalentInjection);
```

Comment:
Could be write in a single line: `nodesEquivalentInjections.computeIfAbsent(node, ls -> new ArrayList<>(2)).add(equivalentInjection);`

Why do you set the capacity to 2?

Reply:
@annetill Is there a reason? I didn't touch your code here.

Reply:
Rectification: it seems to be @zamarrenolm's code. @zamarrenolm can you tell us why the list capacity is at 2 please?

Reply:
Because in a boundary TN, you have 2 EquivalentInjections, one of each IGM.